features/upgrade/etcd/upgrade.feature

```diff
@@ -13,26 +13,26 @@ Feature: basic verification for upgrade testing
   @arm64 @amd64
   Scenario: etcd-operator and cluster works well after upgrade - prepare
     Given I switch to cluster admin pseudo user
-    Given I obtain test data file "admin/subscription.yaml"
-    When I run the :create client command with:
-      | f | subscription.yaml |
-    Then the step should succeed
-    When I use the "openshift-operators" project
-    Then status becomes :running of exactly 1 pods labeled:
-      | name=etcd-operator-alm-owned |
-    When I use the "default" project
-    Given I obtain test data file "admin/etcd-cluster.yaml"
-    When I run the :create client command with:
-      | f | etcd-cluster.yaml |
-    Then the step should succeed
-    #bugzilla - id=1979550
-    #Then status becomes :running of exactly 3 pods labeled:
-    #  | etcd_cluster=example |
-    When I run the :get admin command with:
-      | resource | EtcdCluster |
-    Then the step should succeed
-    And the output should match:
-      | example.* |
+    When I use the "openshift-etcd" project
+    And status becomes :running of 3 pods labeled:
+      | app=etcd |
+    Given evaluation of `@pods[0].name` is stored in the :etcdpod clipboard
+    When I run the :rsh client command with:
+      | pod         | <%= cb.etcdpod %> |
+      | command     | etcdctl           |
+      | command_arg | endpoint          |
+      | command_arg | health            |
+    Then the output should contain 3 times:
+      | is healthy: successfully committed proposal |
+    #login in etcd member pod, and add data in the form of key - value
+    When I run the :rsh client command with:
+      | pod         | <%= cb.etcdpod %> |
+      | command     | etcdctl           |
+      | command_arg | put               |
+      | command_arg | data-ocp26206     |
+      | command_arg | value-of-OCP26206 |
+    Then the output should contain:
+      | OK |
```

Comment:
it will be better add etcd-operator pods checking in ns: openshift-etcd-operator, and also etcd cluster operator checking(oc get co|grep etcd, check the status: True False False )